documentation.html

```diff
@@ -155,6 +156,328 @@ <h3 class="component-subtitle">Outlined Alert</h3>
           </iframe>
         </div>
       </section>
+
+      <!------------- AVATAR COMPONENT ------------->
+
+      <section class="component-section" id="avatar">
+        <h1 class="component-title">Avatar</h1>
+
+        <p class="component-text text-sm">
+          The Avatar component is used to represent a user, and displays the
+          profile picture
+        </p>
+
+        <h3 class="component-subtitle">Basic Avatar</h3>
+
+        <div class="component-display flex-row justify-center flex-wrap">
+          <img
+            src="https://www.industrialempathy.com/img/remote/ZiClJf-1920w.jpg"
+            alt="Avatar"
+            class="avatar"
+          />
+        </div>
+
+        <div class="component-code">
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=seti&wt=none&l=auto&width=766&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%250A%2520%2520%2520%2520src%253D%2522%255Bimage%2520source%255D%2522%250A%2520%2520%2520%2520alt%253D%2522Avatar%2522%250A%2520%2520%2520%2520class%253D%2522avatar%2522%250A%252F%253E"
+            style="
+              width: 766px;
+              height: 167px;
+              border: 0;
```

Comment:
try not to use inline styling,

Reply:
Those are styles that come with the carbon code snippet when copying the `iframe`